signup.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Both in here and in "signin.html" you wrote `id="sign"` (line 29) - if it's repeating - use class instead